### ISSUE TITLE
sf1_2010_detail_variables.json missing

### ISSUE THREAD

**Issue by michaelcollinsedgov** [jtleider/censusdata#31]
I'm new to the package, but have gotten a FileNotFoundError one two completely different machines.  I checked and the file isn't there, but there is a file called `sf1_2010_variables.json`.

Is the file missing, or misnamed, or something else?

**Comment by jtleider** [comment]
There isn't supposed to be a file named sf1_2010_detail_variables.json. Can you specify the code you're running that's leading to this error?

**Comment by michaelcollinsedgov** [comment]
```
---------------------------------------------------------------------------
FileNotFoundError                         Traceback (most recent call last)
<ipython-input-6-3ed59b5ca2a2> in <module>
----> 1 sample_sf1 = censusdata.search('sf1',2010,'concept','gender')

~/.local/lib/python3.7/site-packages/censusdata/variable_info.py in search(src, year, field, criterion, tabletype)
    169                 raise ValueError(u'Unknown table type {0}!'.format(tabletype))
    170         topdir, filename = os.path.split(__file__)
--> 171         with open(os.path.join(topdir, 'variables', '{0}_{1}_{2}_variables.json'.format(src, year, tabletype))) as infile:
    172                 allvars = infile.read()
    173         allvars = json.loads(allvars)['variables']

FileNotFoundError: [Errno 2] No such file or directory: '/home/mcollins/.local/lib/python3.7/site-packages/censusdata/variables/sf1_2010_detail_variables.json'
```

**Comment by jtleider** [comment]
You're right, that's an error in the code. I will put that on my list to fix. In the meantime as a temporary fix you can just rename sf1_2010_variables.json to sf1_2010_detail_variables.json.

**Comment by jtleider** [comment]
Fixed in version 1.14. Thank you for pointing this issue out.
